package.json

```diff
@@ -40,7 +40,8 @@
     "update-versions": "yarn workspace botbuilder-repo-utils update-versions"
   },
   "devDependencies": {
-    "@azure/ms-rest-js": "1.9.0",
+    "@azure/logger": "^1.0.2",
+    "@azure/ms-rest-js": "1.9.1",
```

Comment:
This also should likely be reconciled with whatever package(s) it's used in. Why was this updated?

Reply:
For consistency with other uses of the same dependency. We can revert it if you deem it so